src/styles/price.css

```diff
@@ -0,0 +1,17 @@
+.price {
+  display: flex;
+  justify-content: space-between;
+  line-height: 18px;
+  margin-top: 22px;
```

Comment:
'price' is your block. We can not give external styling like margin or positioning to block. You can create an element and then give it the external styling.
refer checklist number 6